### PR TITLE
added debug message

### PR DESCRIPTION
**Pull request by matthewalbani**
added debug. and fixed indentation.

to use debug  `export SOLANO_CLIENT_DEBUG=true`

### CODE REVIEW

File: lib/tddium_client.rb [hunk 1]
```diff
@@ -14,15 +14,15 @@ module TddiumClient
...
-             Errno::ETIMEDOUT,
-             Timeout::Error,
-             OpenSSL::SSL::SSLError,
-             OpenSSL::SSL::Session::SessionError,
-             HTTPClient::TimeoutError,
-             HTTPClient::BadResponseError,
-             SocketError ]
-
-  module Error
+   Errno::ETIMEDOUT,
```
Comment (semipermeable) [inline lib/tddium_client.rb]: nit: now the indentation here looks strange.

Comment (matthewalbani) [inline lib/tddium_client.rb]: moved that part back

**Comment by matthewalbani** [comment]
looks like 

```
TDDIUM CLIENT DEBUG:
METHOD: post
PATH: https://ec2-52-23-250-250.compute-1.amazonaws.com/1/sessions
HEADERS: {"Content-Type"=>"application/json", "X-Tddium-Api-Key"=>"470c5a75b8a5dc5536bb7b4a75b89313b228a263", "X-Tddium-Client-Version"=>"tddium_client-0.6.0;solano-1.28.0"}
BODY: {"commits_encoded":"kYWmYXV0aG9ygqVlbWFpbLlNYXR0aGV3LmphbGJhbmlAZ21haWwuY29tpG5h\nbWWuTWF0dGhldyBBbGJhbmmpY29tbWl0dGVygqVlbWFpbLlNYXR0aGV3Lmph\nbGJhbmlAZ21haWwuY29tpG5hbWWuTWF0dGhldyBBbGJhbmmkZGF0Zc5W+wwC\nomlk2gAoMzBlZmU5MWM4OTUyNzEyNTc3NDJmY2IxMzIwOWIxZWZkODcwZWJj\nZKdzdW1tYXJ5rnJlbW92ZSByYXRjaGV0\n","cache_control_encoded":"gqdHZW1maWxl2gAoNjFkNjcxMWEzNzc2OTVjNjA0YjRjYTUyZWZlNjkyYjgz\nODI1NzY0N6xHZW1maWxlLmxvY2vaACgxZmQ2NTJhY2IzMDA3NWFlMThiMmQ3\nMGM4YzBiZTNkZWJjODc0YWMy\n","cache_save_paths_encoded":"wA==\n","raw_config_file":"cnVieV92ZXJzaW9uOiAiMi4xLjIiCmJ1bmRsZXJfdmVyc2lvbjogMS4xMC42\nCmNvdmVyYWdlOgogIHZlcnNpb246IDIKICBlbmFibGVkOiB0cnVlCnRlc3Rz\nOgogIC0gZWNobyAkUEFSQUxMRUxfVEVTVF9HUk9VUFMKICAtIGVjaG8gJFRF\nU1RfRU5WX05VTUJFUgogIC0gZWNobyAiaW0gYSBwYXRjaCIKICAtIGVjaG8g\nImltIG5vdCBpbiB1cHN0cmVhbSIKICAtIGVjaG8gIndpdGggc25hcHNob3Qi\nCg==\n","env":{},"suite_id":53,"xid":"74e4c89086c9a49f"}
```

**Comment by matthewalbani** [comment]
@bkendzior @twestberg fyi, you guys might find this helpful

**Comment by matthewalbani** [comment]
made it more modular and added
``
TDDIUM CLIENT DEBUG:
result: {"status"=>0, "snap_commit"=>"179dae8c22180bab8bc4ae75cb539dee10c56456"}
```

**Pull request merged**